SU-HSDO/suhumsci
# shs-101: adjusts hero font sizes so theyre smaller than h1

## Summary
This adjusts the hero title font size by removing the breakpoint overrides that increased the font size. 

## Steps to Test
1. On both Traditional and Colorful, create a page. Add a spotlight slider and fill in the forms. Save.
2. View the page and across breakpoints, ensure the hero's title is a smaller `font-size` than the page's `h1`. This is to ensure visual hierarchy. 

## Notes
Will change base branch once release is created.

## Screenshots
![Screen Shot 2023-02-13 at 10 59 41 AM](https://user-images.githubusercontent.com/8405274/218508907-99e2af08-4783-411e-a9d6-7c662ec77c20.png)
